README.md

```diff
@@ -294,7 +295,29 @@ For **∅ or more** and **1 or more** element(s) of the same type you can use th
 Union types are denoted with the pipe symbol, `|`:
 
 ```js
-(userInput: String|Number) => String|Number
+(userInput: String | Number) => String | Number
+```
+
+### Negated Types
+
+It is sometime easier and more informative to delimit a type by defining what it's not. The negation operator lets you exclude by substracting from `Any`.
+
+```js
+JSON::parse(String, reviver: Function)
+  => Boolean | Number | String | Object | Array | null,
```

Comment:
@ericelliott there's a problem with this example: `Array` and `Object` are not precise enough.
These objects/arrays must contain JSON values: it's recursive.

The example should be replaced or updated.